lib/util/string.ts

```diff
@@ -45,3 +45,13 @@ export function uniqueStrings(
 ): boolean {
   return elements.indexOf(element) === index;
 }
+
+export function looseEquals(
+  a: string | null | undefined,
+  b: string | null | undefined
+): boolean {
+  if (!(a && b)) {
+    return a === b;
+  }
```

Comment:
@viceice OK to do a raw equals like this if a or b is not truthy?

Reply:
so `null === undefined` will return `false`